src/protocols/aeternity/libs/AeScan.ts

```diff
@@ -37,7 +37,7 @@ export class AeScan extends ProtocolExplorer {
     return (endpoint) ? `${this.explorerUrl}/${endpoint}/${hash}` : undefined;
   }
 
-  override prepareUrlForAccount(address: string) {
-    return `${this.explorerUrl}/accounts/${address}`;
+  override prepareUrlForAccount(addressOrName: string) {
+    return this.prepareUrlForHash(addressOrName) || `${this.explorerUrl}/accounts/${addressOrName}`;
```

Comment:
When do we call this function with a name?

Reply:
In some cases `recipientId` ([here](https://github.com/superhero-com/superhero-wallet/pull/2678/files#diff-7555767fde85bbc840175b6d9e1eaa7e9459580fe4cc884750809f322c9986e2R203))  is a name address (`nm_123`)

![image](https://github.com/superhero-com/superhero-wallet/assets/49337371/3a81777f-2f76-4e4b-b0ff-f1225e9483e4)